doc/source/whatsnew/v0.18.2.txt

```diff
@@ -76,6 +76,7 @@ Other enhancements
 
 - The ``pd.read_csv()`` with ``engine='python'`` has gained support for the ``decimal`` option (:issue:`12933`)
 - The ``pd.read_csv()`` with ``engine='python'`` has gained support for the ``na_filter`` option (:issue:`13321`)
+- The ``pd.read_csv()`` with ``engine='python'`` has gained support for the ``compact_ints`` and ``use_unsigned`` options (:issue:`13323`)
```

Comment:
hmm, I agree here. Since we are deprecating this, I think that no need to announce it for python engine support.